SimPEG/Directives.py

```diff
@@ -1081,3 +1076,50 @@ def updateOpt(self):
             JtJdiag += JtJ
 
         self.opt.JtJdiag = JtJdiag
+
+
+class ProjSpherical(InversionDirective):
```

Comment:
would you mind spelling this out `ProjectSpherical`?

Reply:
```ProjectSphericalBounds```